package.json

```diff
@@ -10,6 +10,7 @@
     "build:errors-and-warnings": "sass --no-source-map src/checka11y.scss ./checka11y.css --style=compressed && sass --no-source-map ./checka11y.css ./checka11y.css --style=expanded && npx postcss checka11y.css --use autoprefixer --replace --no-map",
     "build:errors-only": "sass --no-source-map src/errors/checka11y-errors.scss ./checka11y-errors.css --style=compressed && sass --no-source-map ./checka11y-errors.css ./checka11y-errors.css --style=expanded && npx postcss checka11y-errors.css --use autoprefixer --replace --no-map",
     "build:warnings-only": "sass --no-source-map src/warnings/checka11y-warnings.scss ./checka11y-warnings.css --style=compressed && sass --no-source-map ./checka11y-warnings.css ./checka11y-warnings.css --style=expanded && npx postcss checka11y-warnings.css --use autoprefixer --replace --no-map",
+    "build:minify": "npm run build && node minify.js",

```

Comment:
Maybe this script could just build the minified versions, and the `npm run build` script we could add `&& npm run build:minify` to it?

Reply:
Hmm I think that would be fine. The only issue I see is that the `npm run build` command will always minify CSS if it has `&& npm run build:minify` appended at the end. Which is "ok" if we want the generic `npm run build` to always minify. My thoughts with `build:minify` was to provide a new command that runs the normal `npm run build` and then also minifies the stylesheet(s) since that is a "new" option.

Reply:
Yeah that sounds fine, let's go with what you suggest! 🙂

Reply:
Ok deal :)